lib/bolt/pal.rb

```diff
@@ -635,6 +635,36 @@ def list_modules
       end
     end
 
+    # Return information about a module.
+    #
+    # @param name [String] The name of the module.
+    # @return [Hash]
+    #
+    def show_module(name)
+      name = name.tr('-', '/')
+
+      data = in_bolt_compiler do |_compiler|
+        mod = Puppet.lookup(:current_environment).module(name.split(%r{[/-]}, 2).last)
```

Comment:
I think an integration test would be useful for this, especially since it relies on the keys we get back from this lookup + pass through.

Reply:
@lucywyman Added a few unit tests and a single integration test. :)